test/successful-directories.test.ts

```diff
@@ -780,6 +780,52 @@ describe('searchPlaces can include subdirectories', () => {
   });
 });
 
+describe('directories are not treated as files', () => {
```

Comment:
I think a more verbose description or multi-sentence comment would be helpful for explaining this test case. It takes some very close reading to realize that there's a directory named `package.json`, and that's the focus of the test. Could we call out what's going on here?

Reply:
Pushed a commit. Is that clearer?

Reply:
Yeah, looks good 🙇